libs/client-api/src/v2/actor.rs

```diff
@@ -521,7 +531,9 @@ impl WorkspaceControllerActor {
     while let Some(res) = stream.next().await {
       if cancel.is_cancelled() {
         sync_trace!("remote receiver loop cancelled");
-        break;
+        return Err(DisconnectedReason::MessageLoopEnd(
+          "remote receiver loop cancelled".into(),
```

Comment:
Same here: cancel is called by the client on explicit disconnection. It's not an error.